.circleci/config.yml

```diff
@@ -135,7 +135,8 @@ run_oss_benchmark: &run_oss_benchmark
   - run:
       name: Run OSS Benchmark
       command: |
-        python benchmarks/oss.py --check_regression --world_size 4 --reference_speed 660 --reference_memory 1120 --reference_loss 0.023
+        python benchmarks/oss.py --world_size 4 --epochs 2
+        python benchmarks/oss.py --check_regression --world_size 4 --optim_type oss_sharded_ddp --reference_speed 660 --reference_memory 930 --reference_loss 0.023
```

Comment:
same loss, just a tighter reference memory, 930MB vs. 1120 for the same config. The idea is that we're testing OSS+ShardedDDP, which has a lower memory consumption actually, this was not tested properly. We were checking for the worst case, which degraded slightly this afternoon